src/main/java/sirius/pasta/tagliatelle/TagliatelleController.java

```diff
@@ -53,71 +55,76 @@ public void onError(WebContext webContext, HandledException error) {
     /**
      * Renders a list of all known tags.
      *
-     * @param ctx the request to process
+     * @param webContext the request to process
      */
     @Routed("/system/tags")
     @Permission(PERMISSION_SYSTEM_TAGS)
-    public void overview(WebContext ctx) {
-        Collection<Macro> macros = context.getParts(Macro.class);
+    public void overview(WebContext webContext) {
+        Collection<Macro> macros = context.getParts(Macro.class)
+                                          .stream()
+                                          .sorted(Comparator.comparing(Macro::getName))
+                                          .collect(Collectors.toList());
         Collection<String> builtIns = context.getParts(TagHandlerFactory.class)
                                              .stream()
+                                             .sorted(Comparator.comparing(TagHandlerFactory::getName))
```

Comment:
Move sorting down, as last step before `.collect`? We will have strings there anyway.